outputs.tf

```diff
@@ -42,6 +42,11 @@ output "bucket_storage_class" {
   value       = local.bucket_storage_class
 }
 
+output "bucket_region" {
+  description = "Bucket region"
```

Comment:
Update to this since the module supports creating cross regional and single site buckets too:

> Bucket region if you created a regional bucket